packages/d2mini/src/operators/topK.ts

```diff
@@ -34,9 +34,8 @@ export function topK<
     const reduced = stream.pipe(
       reduce((values) => {
         // `values` is a list of tuples, first element is the value, second is the multiplicity
-        const consolidated = new MultiSet(values).consolidate()
-        const sortedValues = consolidated
-          .getInner()
+        const consolidated = LazyMultiSet.fromArray(values).consolidate()
```

Comment:
Why creating a lazy multiset to then consolidate it (which materializes it) to then immediately copy it into an array. Seems like a lot of overhead for something we could do by reducing once over the original array (i.e. loop over `values` and consolidate it into an array of consolidated values).